deploy/spoke/role.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: open-cluster-management:spoke
```

Comment:
Let's think carefully about using the name `spoke` here. `agent` might be a better alternative. Do you have a preference on this detail, @mdelder ?

Reply:
how about `registration-agent`? I think this more specific